packages/terra-search-field/src/terra-dev-site/doc/example/SearchFieldDisabled.jsx

```diff
@@ -1,11 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
-import SearchFieldExampleTemplate from 'terra-search-field/lib/terra-dev-site/doc/example/SearchFieldExampleTemplate';
+import SearchField from 'terra-search-field/lib/SearchField';
 
-const SearchFieldDisabled = () => (
-  <SearchFieldExampleTemplate
-    isDisabled
-  />
-);
+const INVALID_MESSAGE = 'The default minimum search length is 2.';
+
+const SearchFieldDisabled = () => {
+  const [searchText, setSearchText] = useState('');
+
+  const message = searchText.length >= 2 ? `Search text: ${searchText}` : INVALID_MESSAGE;
+
+  return (
+    <div>
+      <p>{message}</p>
+      <SearchField isDisabled onSearch={setSearchText} onInvalidSearch={setSearchText} />
```

Comment:
I dont think so we require the code changes for this example. as it is to show the disabled control.

Reply:
Earlier this example was using a template file which was not descriptive. Hence moved it to a new file so consumers can get a clear idea.

Reply:
That makes sense.